components/auth/Login.tsx

```diff
@@ -122,23 +122,23 @@ const Login: FC = () => {
     const mode =
       query.mode || qs.parse(location.search, { ignoreQueryPrefix: true }).mode;
 
-    if (mode === 'issuer') {
-      const [err, issuer] = await signIn({
+    if (mode === 'issuer' || mode === 'user') {
+      const [err, user] = await signIn({
         userInfo: {
           email: account,
           password,
         },
       });
-      if (!issuer || issuer.length === 0) {
+      if (!user || user.length === 0) {
         notify.error({ msg: err || '此帳號並不存在' });
-      } else if (!issuer[0].isIssuer) {
+      } else if (!user[0].isIssuer) {
```

Comment:
I think this condition should be like `else if (mode ==== 'issuer' && !user[0].isIssuer)` ?